pyaedt/application/Variables.py

```diff
@@ -156,14 +157,17 @@ def __getitem__(self, item):
                 if variable in key_string:
                     found_variable = True
                     break
-            assert found_variable, "Input string {} is not a key of the data dictionary.".format(variable)
+            if not found_variable:
+                raise KeyError("Input string {} is not a key of the data dictionary.".format(variable))
             data_out._data[variable] = self._data[key_string]
             data_out._header.append(variable)
         return data_out
 
     @pyaedt_function_handler()
     def __add__(self, other):
-        assert self.number_of_columns == other.number_of_columns, "Inconsistent number of columns"
+        if self.number_of_columns != other.number_of_columns:
+            raise ValueError("Inconsistent number of columns.")
```

Comment:
```suggestion
            raise ValueError("Number of columns is inconsistent.")
```
Message should be a complete sentence.